### ISSUE TITLE
Digitizing on DEM (and surfaces in general)

### ISSUE THREAD

**Issue by andrea-bistacchi** [andrea-bistacchi/PZero#39]
I've found an interesting discussion:

https://discourse.vtk.org/t/how-to-draw-lines-on-surface/10905

**Comment by gbene** [comment]
Yes i think we should look at the code of the [3d slicer](https://github.com/Slicer/Slicer/tree/main/Modules/Loadable/Markups) at the end of the thread to see what they are doing. While experimenting with the different widgets for the new 2D view I've stumbled upon this: [https://vtk.org/doc/nightly/html/classvtkImageTracerWidget.html#details](https://vtk.org/doc/nightly/html/classvtkImageTracerWidget.html#details) that can be actually used on any generic 3D prop. I've tried it and it seemed pretty quick and well functioning.
The only downside to this widget is that it is not clear if the mapped keys are customizable or not (I think not)

**Comment by andrea-bistacchi** [comment]
merged int #84
